runtime/calamari/src/lib.rs

```diff
@@ -297,6 +304,10 @@ impl Contains<RuntimeCall> for BaseFilter {
             | RuntimeCall::XTokens(orml_xtokens::Call::transfer {..}
                 | orml_xtokens::Call::transfer_multicurrencies {..})
             | RuntimeCall::TransactionPause(_)
+            | RuntimeCall::ZenlinkProtocol(_)
+            | RuntimeCall::ZenlinkStableAMM(_)
+            | RuntimeCall::ZenlinkSwapRouter(_)
+            | RuntimeCall::Farming(_)
```

Comment:
Do you think we can drop any of the user callable extrinsics and simplify the system overall